src/metrics_server/Taskfile.yml

```diff
@@ -0,0 +1,57 @@
+version: '3'
+
+vars:
+  SERVER_DIR: '{{joinPath .USER_WORKING_DIR "build/metrics_server"}}'
+
+tasks:
+  clean:
+    desc: Clean metrics server output
+    cmds:
+      - rm -rf "{{.SERVER_DIR}}"
+
+  build:
+    desc: Build the metrics server
+    vars:
+      BUILD_MODE: '{{.BUILD_MODE | default "dev"}}'
+      TARGET_DIR: '{{joinPath .SERVER_DIR .BUILD_MODE}}'
+    cmds:
+      - rm -rf '{{.TARGET_DIR}}'
+      - npx tsc --project '{{.TASKFILE_DIR}}' --outDir '{{.TARGET_DIR}}'
+      - cp '{{joinPath .TASKFILE_DIR "package.json"}}' '{{.TARGET_DIR}}'
+      - cp '{{joinPath .USER_WORKING_DIR "package-lock.json"}}' '{{.TARGET_DIR}}'
+      - cp '{{.TASKFILE_DIR}}/app_{{.BUILD_MODE}}.yaml' '{{.TARGET_DIR}}/app.yaml'
+      - cp '{{.TASKFILE_DIR}}/config_{{.BUILD_MODE}}.json' '{{.TARGET_DIR}}/config.json'
+
+  deploy:dev:
+    desc: Deploy the development metrics server
+    deps: [{task: build, vars: {BUILD_MODE: dev}}]
+    cmds:
+      - gcloud app deploy '{{.TASKFILE_DIR}}/dispatch.yaml' '{{joinPath .SERVER_DIR "dev"}}' --project uproxysite --verbosity info --promote --stop-previous-version
+
+  deploy:prod:
+    desc: Deploy the production metrics server
+    deps: [{task: build, vars: {BUILD_MODE: prod}}]
+    cmds:
+      - gcloud app deploy '{{.TASKFILE_DIR}}/dispatch.yaml' '{{joinPath .SERVER_DIR "prod"}}' --project uproxysite --verbosity info --no-promote --no-stop-previous-version
+
+  start:
+    desc: Start the metrics server locally
+    deps: [build]
+    cmds:
+      - cp '{{.TASKFILE_DIR}}/config_dev.json' '{{.SERVER_DIR}}/config.json'
+      - npx node '{{.SERVER_DIR}}/index.js'
```

Comment:
Do we need `npx` here? Can we just use `node {{.SERVER_DIR}}/index.js`?

Reply:
Removed